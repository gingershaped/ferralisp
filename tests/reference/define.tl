(d x 42) (q x) x
(d y (q x)) y (v y)
(d self (q self)) self
(d L (q (L))) L