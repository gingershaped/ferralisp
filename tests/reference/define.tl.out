x
x
42
y
x
42
self
self
L
(L)